src/actions/GenericActions.js

```diff
@@ -38,6 +38,26 @@ export function getData(
     );
 }
 
+/**
+ * 
+ * @param {*} entity - entity should always be `documentView`
```

Comment:
then why having it as parameter?

Reply:
just for the future development

Reply:
Agile development best practices(1) says that we shall not develop ahead and imagine things because they might never happen. It might be that there will be no future extension. And when and if it will be, we will address it at that time.
At the moment i find this misleading for new developers.
More I would rename the function from "getAttributes" to "getViewAttributes" because that's what precisely is doing.
I will drop that "entity" param because it;s pointless and i will also drop that viewId and rowId checking because if they are null/missing, you would build a wrong endpoint URL anyways.

Also, please note, I am not a javascript developer, so it might be wrong what I am saying.
I would like to hear other oppinions too.
@PabloSichert  @metas-ts  @ottosichert  guys, wdyt?

References:
1. Clean Code: A Handbook of Agile Software Craftsmanship (Robert C. Martin)